easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3_bamtools_includepath.patch

```diff
@@ -0,0 +1,25 @@
+diff -Nru augustus.orig/auxprogs/bam2hints/Makefile augustus/auxprogs/bam2hints/Makefile
```

Comment:
Please include a comment at the top of the patch indicating the reason for including it.